resources/cheatsheet.md

```diff
@@ -34,3 +34,8 @@ Delete everything in a directory with confirmation:
 ```
 rm -iR directory
 ```
+Move a file or Directory:
+```
+mv <directory name> <new directory name>
+mv <file name> <new file name>
+```
```

Comment:
this could be clarified a little. 

What are the two things that the `mv` command can do? how could you make that more clear here?